subprojects/ce-login/celogin/include/CeLogin.h

```diff
@@ -219,19 +223,21 @@ struct AcfUserFields
     uint64_t& expirationTimeParm);
 
 /// @note This function will return failure if called with a V2 ACF
-[[deprecated]] CeLoginRc checkServiceAuthorityAcfIntegrityV1(
+CeLoginRc checkServiceAuthorityAcfIntegrityV1(
     const uint8_t* accessControlFileParm,
     const uint64_t accessControlFileLengthParm,
     const uint64_t timeSinceUnixEpochInSecondsParm, const uint8_t* publicKeyParm,
     const uint64_t publicKeyLengthParm, const char* serialNumberParm,
     const uint64_t serialNumberLengthParm, ServiceAuthority& authorityParm,
     uint64_t& expirationTimeParm);
 
-/** @brief Sanity check an ACF file (excluding password validation)
+/** @brief Validate an ACF file at upload time
  *
- *  This function decodes and verifies several aspects of the provided ACF file.
+ *  This function decodes and verifies several aspects of the provided ACF.
  *  This includes verifying the signature over the ACF, ensuring the ACF is not expired,
- *  checking the serial number fields.
+ *  checking the serial number fields, and validating the replay ID, if it exists.
+ *  Upon successful validation, the output fields are udpated with the values obtained
+ *  from the ACF. THE CALLER MUST PERSIST THE REPLAY ID PROVIDED ON SUCCESS.
```

Comment:
THE CALLER MUST PERSIST THE RESULTING REPLAY ID ON SUCCESS.

resulting replay id?
returned replay id?